packages/calcite-design-tokens/support/token-transformer/styleDictionary/formatter/utils/formatExtraOutput.ts

```diff
@@ -44,7 +44,7 @@ export function formatExtraOutput(
             ? index.media.map(([mediaSchemed, output]) => {
                 const m = ensureIfArray(outputObject[`${output}.${args.platform}`]);
                 return Array.isArray(m)
-                  ? `@media (${mediaSchemed}) {\n\t:root {\n\t\t${m
+                  ? `@media (${mediaSchemed}) {\n\t.calcite-mode-auto {\n\t\t${m
```

Comment:
As a follow up (not in this PR), we shouldn't be using `\n` since it can cause issues on Windows. Is there an issue for tracking the Windows related design token build errors that Kitty ran into? If not, can you please log one?

For reference, you can do this instead:

```js
import { EOL }from "os";

const osAgnosticString = `${EOL}lorem${EOL}ipsum${EOL}`
```